examples/ExchangeOracles/contracts/ExchangeMarket.aes

```diff
@@ -0,0 +1,58 @@
+contract ExchangeMarket =
+
+  record state = {
+    aePrice : int,
+    tokenPrice: int}
+
+  type query_type = string
+  type answer_type = int
+  type oracle_id = oracle(query_type, answer_type)
+  type query_id = oracle_query(query_type, answer_type)
+
+  public stateful function init(_aePrice: int, _tokenPrice: int) = 
+    require(_aePrice > 0 , "The price for the AE must be greater than zero ")
+    require(_tokenPrice > 0 , "The price for the token must be greater than zero")
+    {aePrice = _aePrice,
+      tokenPrice = _tokenPrice}
+    
+    
+  public function getAePrice() : int = state.aePrice
+  public function getTokenPrice() : int = state.tokenPrice
+
+  function createQuery(oracle: oracle_id, question: query_type, qfee: int, qttl: int, rttl: int) : query_id =
+    require(qfee > 0 , "Query fee must be greater than zero")
+    require(qttl > 0 , "Query TTL must be greater than zero")
+    require(rttl > 0 , "Relative TTL must be greater than zero")
+    Oracle.query(oracle, question, qfee, RelativeTTL(qttl), RelativeTTL(rttl))
+
+  function createAePriceQuery(oracle: oracle_id, qfee: int, qttl: int, rttl: int) : query_id =
```

Comment:
maybe 'qfee' should be qFee.. in camel case variant. same for 'qttl' and 'rttl'

Reply:
>  in camel case variant.

I think in the functional languages like Sophia snake_case is the preferred one (I'm trying to get used to it as well :D).

Reply:
I agree with @mradkov